commands/motor/MotorPositionConstant.java

```diff
@@ -61,39 +61,44 @@ protected void initialize() {
 		catch (InvalidSensorException e) {
 			cancel();
 			if (fallbackCommand != null) {
-				fallbackCommand.start();
+				fallbackCommand.schedule();
 			}
 			return;
 		}
 	}
 
 	@Override
-	protected void execute() {
+	public void execute() {
 		if (motor.checkSensorException() != null) {
 			cancel();
 			if (fallbackCommand != null) {
-				fallbackCommand.start();
+				fallbackCommand.schedule();
 			}
 		}
 	}
 
 	@Override
-	protected boolean isFinished() {
+	public boolean isFinished() {
 		if (endOnArrival) {
 			return motor.onTarget();
 		}
 		return false;
 	}
 
 	@Override
-	protected void end() {
+	public void end(boolean interrupted) {
 		motor.disableMotionController();
 		// Please just die in a hole and never come back
-		if (fallbackCommand != null && fallbackCommand.isRunning()) {
+		if (fallbackCommand != null && fallbackCommand.isScheduled()) {
 			fallbackCommand.cancel();
 		}
 	}
-
+	
 	@Override
-	protected void interrupted() {}
```

Comment:
Again, it seems to me that this change will alter what happens when the command is interrupted, and I'm not sure if that was intentional.